apps/designer/app/designer/features/style-panel/shared/style-info.ts

```diff
@@ -54,6 +54,11 @@ export const getStyleSource = (
       return "local";
     }
   }
+  for (const info of styleValueInfos) {
```

Comment:
Having that `getStyleSource` of multiple properties is used in 1 place, may be remove `styleValueInfos` array from argument. As I'm not getting logic why do we return 'local' if any of the info is local,
then do the same with remote? Or at least comment on the function.

Reply:
It's trade off for controls with multiple properties. We need to get at least one match